idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/settings/InMemorySettings.kt

```diff
@@ -72,7 +76,7 @@ class InMemorySettings(project: Project) {
             flatPackage = flatPackage,
             useExplicitMode = useExplicitMode,
             addTrailingComma = addTrailingComma,
-
+            fileIndent = fileIndent.or(project.getIndentSize()).toInt(),
```

Comment:
```suggestion
            fileIndent = fileIndent,
```

Looks we don't need this `or`?

Reply:
make sense, i forgot remove this one